CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+* add functions as UDF input types ([#6955](https://github.com/confluentinc/ksql/pull/6955)) ([1b39ab5](https://github.com/confluentinc/ksql/commit/1b39ab56360612d385554d61c137814655dbf4b9))
+* add functions to support use of timestamp data type ([#6852](https://github.com/confluentinc/ksql/pull/6852)) ([2cee618](https://github.com/confluentinc/ksql/commit/2cee618ef8ca8a24768e0f7b1b88ccd2e5364248))
+* add lambda syntax to grammar ([#6868](https://github.com/confluentinc/ksql/pull/6868)) ([dd3f365](https://github.com/confluentinc/ksql/commit/dd3f365eba725c77cca63f9196dac174394ecb58))
+* add standard deviation udf ([#6845](https://github.com/confluentinc/ksql/pull/6845)) ([f2fdbb3](https://github.com/confluentinc/ksql/commit/f2fdbb389d06aafc808493fe009719f202351728))
+* adding TypeContext for Lambdas ([#6989](https://github.com/confluentinc/ksql/pull/6989)) ([1538b3a](https://github.com/confluentinc/ksql/commit/1538b3ad7975adbc1ad0a452a7538ad7f3f748ba))
+* Adds ability to bypass cache for pull queries ([#6891](https://github.com/confluentinc/ksql/pull/6891)) ([4b3bc96](https://github.com/confluentinc/ksql/commit/4b3bc96830fbea8051cfaf4e9d27c4399032813f))
+* Adds table scans to pull queries ([#6726](https://github.com/confluentinc/ksql/pull/6726)) ([ae14fca](https://github.com/confluentinc/ksql/commit/ae14fca76089b85653de15003aead0dae4d2a5de))
+* Allows pull queries with generic WHERE clauses ([#6939](https://github.com/confluentinc/ksql/pull/6939)) ([c3fe8a1](https://github.com/confluentinc/ksql/commit/c3fe8a1aaf17c40f1205374f13a633ca6ebdbe19))
+* display precision and scale when describing DECIMAL columns ([#6872](https://github.com/confluentinc/ksql/pull/6872)) ([fb89998](https://github.com/confluentinc/ksql/commit/fb89998ede01922a4741448f37c7064aa966147e))
+* enable decimal for Protobuf ([#6884](https://github.com/confluentinc/ksql/pull/6884)) ([6b5877c](https://github.com/confluentinc/ksql/commit/6b5877c2910023263421b247824af4eadb7db640))
+* Make pull queries streamed asynchronously ([#6813](https://github.com/confluentinc/ksql/pull/6813)) ([b69e3f8](https://github.com/confluentinc/ksql/commit/b69e3f8e459c3d0a055ec0738ae71e58bda7b28c))
+* make UDAFs configurable and remove limit on COLLECT_LIST/SET ([#6851](https://github.com/confluentinc/ksql/pull/6851)) ([63ae169](https://github.com/confluentinc/ksql/commit/63ae169ab753aab980f0ee4cbf52fbe34291418d))
+* Move WHERE clause handling for pull queries into logical plan ([#6814](https://github.com/confluentinc/ksql/pull/6814)) ([fa6758a](https://github.com/confluentinc/ksql/commit/fa6758af2f46594bfdf8f18a6d59a9b40e62e1d0))
+* new and initialize commands for migration tool ([#6988](https://github.com/confluentinc/ksql/pull/6988)) ([8bdb09a](https://github.com/confluentinc/ksql/commit/8bdb09a2751365966cf5a4a1436276d7a3e0d9e1))

```

Comment:
Let's remove this from the changelog -- it's a piece of the migrations tool feature that's shipping with 0.17, not useful on its own.